code/processes/wdb.q

```diff
@@ -232,14 +232,18 @@ doreload:{[pt]
 	];
 	};
 
-endofdaysortdate:{[dir;pt;tablist]
+// set .z.zd to control how data gets compressed
+setcompression:{[compression] if[3=count compression;.lg.o[`compression;$[compression~16 0 0;"resetting";"setting"]," compression level to (",(";" sv string compression),")"];.z.zd:compression]}
```

Comment:
could you split this across lines? 

you could also add a function of 
resetcompression:{setcompression 16 0 0}

which might be nice